modules/FeatureEngineering.py

```diff
@@ -71,6 +71,11 @@ def calculate_abs_speed_deviation(self,row):
     # Yaseen
     def calculate_abs_acceleration_deviation(self,row):  # Use only positive values from lonAcceleration
         track_id = int(row["trackId"])
+        df_tracks = self.data.tracks_df[self.data.tracks_df["tracID"]==track_id]
+        positive_tracks = df_tracks[df_tracks["lonAcceleration"] > 0] 
+        complement = positive_tracks["lonAcceleration"].mean()
+        result = abs(positive_tracks[positive_tracks["lonAcceleration"]-complement]).count()
+        return (result/positive_tracks.count()) 
```

Comment:
you can use .mad() function to get mean absolute deviation 
try
 return positive_tracks["lonAcceleration"].mad()